packages/communication-react/review/communication-react.api.md

```diff
@@ -891,11 +893,21 @@ export interface JumpToNewMessageButtonProps {
 export const lightTheme: PartialTheme & CallingTheme;
 
 // @public
-export const LocaleContext: React_2.Context<ILocale>;
+export interface Locale {
```

Comment:
nitty nit: Do we intend to include stuff other than strings in Locale? If so, might be cleaner to bundle all strings inside a field, with a shorter nam for the fields!

```ts
export interface Locale {
    strings: ComponentStrings;
}

export interface ComponentStrings {
  cameraButton: CameraButtonStrings;
  endCallButton: EndCallButtonStrings;
  // ...
}

// ... usage
 const locale.strings.cameraButton.onText
```

This is a nitty nit, so choose based on taste ;)